hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/storage/TestContainerCommandsEC.java

```diff
@@ -349,9 +349,16 @@ private static byte[] getBytesWith(int singleDigitNumber, int total) {
   @MethodSource("recoverableMissingIndexes")
   void testECReconstructionCoordinatorWith(List<Integer> missingIndexes)
       throws Exception {
-    testECReconstructionCoordinator(missingIndexes);
+    testECReconstructionCoordinator(missingIndexes, 3);
   }
 
+  @Test
+  void testECReconstructionWithPartialStripe()
```

Comment:
Hi @umamaheswararao, the fix looks good.
However, seems the test here is not covering the bug.
Could you please take another look?

Reply:
<img width="1717" alt="Screen Shot 2023-02-08 at 8 39 49 AM" src="https://user-images.githubusercontent.com/1719507/217594332-898765cf-c936-4e23-b7c7-d9df5c6b1378.png">

It is failing when I run this test. 
Interestingly I noticed that, when I run the tests together with testECReconstructionCoordinatorWithMissingIndexes135, it passes. Can you try running this test alone?

Reply:
> It is failing when I run this test. Interestingly I noticed that, when I run the tests together with testECReconstructionCoordinatorWithMissingIndexes135, it passes. Can you try running this test alone?

Same. Maybe there is interference among the tests here?

Reply:
I figured out the problem. Test indeed tests the behavior, however because of using the inputchunks array which was might created by other tests, number of bytes written to file was more than one chunk. So, it's not meeting the expected situation when ran with other tests. If you run this test alone, it will consistently fail as input chunk initialization will happen as expected, that is just 1 chunk. I will update the patch shortly.